be/src/runtime/data_stream_sender.cpp

```diff
@@ -497,7 +503,7 @@ Status DataStreamSender::prepare(RuntimeState* state) {
     std::shuffle(_channel_indices.begin(), _channel_indices.end(), std::mt19937(std::random_device()()));
 
     _bytes_sent_counter = ADD_COUNTER(profile(), "BytesSent", TUnit::BYTES);
-    _uncompressed_bytes_counter = ADD_COUNTER(profile(), "UncompressedBytes", TUnit::BYTES);
+    _bytes_uncompressed_counter = ADD_COUNTER(profile(), "BytesUncompressed", TUnit::BYTES);
```

Comment:
Why change the profile name?

Reply:
So "Bytes" related metrics can be grouped together. Right now, "BytesSent" and "UncompressedBytes" are quite far away to each other in profile text.

Reply:
Is that okay? Or is there any issue of compability that this name can not be changed?